x/observer/keeper/keeper_utils.go

```diff
@@ -32,7 +32,17 @@ func (k Keeper) CheckIfFinalizingVote(ctx sdk.Context, ballot types.Ballot) (typ
 }
 
 // IsAuthorized checks whether a signer is authorized to sign , by checking their address against the observer mapper which contains the observer list for the chain and type
+// It also checks if the signer is a validator and if they are not tombstoned
 func (k Keeper) IsAuthorized(ctx sdk.Context, address string, chain *common.Chain) bool {
+	isPresentInMapper := k.IsObserverPresentInMappers(ctx, address, chain)
+	isTombstoned, err := k.IsOperatorTombstoned(ctx, address)
+	if err != nil || isTombstoned {
+		return false
+	}
+	return isPresentInMapper && !isTombstoned
```

Comment:
```suggestion
	return isPresentInMapper
```
Already checked above

Reply:
Just want to keep the statement to make it more readable